src/Driver.hpp

```diff
@@ -63,6 +61,79 @@ namespace dvl_teledyne
          * Throws std::runtime_error if an error is reported by the device
          */
         void readConfigurationAck(base::Time const& timeout = base::Time::fromSeconds(1));
+
+        /** Configures the number of bottom-track pings per data ensemble.*/
+        void setBottomTrackPingsPerEnsemble(int bottomTrackPingsPerEnsemble);
+
+        /** Configures the maximum tracking depth (10 - 65535 dm).*/
+        void setMaximumTrackingDepth(float maximumTrackingDepth);
+
+        /** Configures the serial port control settings (stopBits : 1 or 2).*/
+        void setSerialPortControlSettings(BAUDRATE baudrate, PARITY parity, int stopBits);
+
+        /** Configures wether ensemble and ping cycling are handled
+         * automatically or manually, wether data output is sent in binary or
+         * hex-ASCII format, and wether data is sent via the serial interface or not.
+         */
+        void setFlowControlSettings(bool automaticEnsembleCycling,
+                                    bool automaticPingCycling,
+                                    bool binaryDataOutput,
+                                    bool enableSerialOutput,
+                                    bool enableDataRecording);
+
+        /** Configures the correction between the heading reference and beam 3 of the
+         * DVL (-17999 - 18000 hundreths of a degree).
+         */
+        void setHeadingAlignment(float headingAlignment);
+
+        /** Same as setHeadingAlignment but corrects for electrical and magnetic bias
+         * instead of physikal misalignment.*/
+        void set_e_headingBias(float e_headingBias);
+
+        /** Configures the waters salinity (0 - 40 parts per thousand).*/
+        void setSalinity(int salinity);
+
+        /** Configures the sensor sources (fixed manual value, internal sensor, external sensor).*/
+        void setSensorSourceSettings(SENSORSOURCE speedOfSoundSource,
+                                     SENSORSOURCE depthSource,
+                                     SENSORSOURCE headingSource,
+                                     SENSORSOURCE pitchAndRollSource,
+                                     SENSORSOURCE salinitySource,
+                                     SENSORSOURCE temperatureSource);
+
+        /** Configures the minimum interval between ensemble acquisition (0 - 24h59'59.99").*/
+        void setTimePerEnsemble(const base::Time& timePerEnsemble);
+
+        /** Configures the minimum time between pings (0 - 59'59.99").*/
+        void setTimeBetweenPings(const base::Time& timeBetweenPings);
+
+        /** Configures the number of depth cells (1 - 255).*/
+        void setNumberOfDepthCells(int numberOfDepthCells);
+
+        /** Configures the number of pings per data ensemble during water profiling (0 - 16384).*/
+        void setPingsPerEnsemble(int pingsPerEnsemble);
+
+        /** Configures the height of one measurement cell (10 - 800 cm).*/
```

Comment:
Same comment than for setHeadingAlignment

```suggestion
        /** Configures the height of one measurement cell
         * @param depthCellSize the size of a depth cell in meters. Must be between 0.01 and 8.
         */
```

Reply:
Applied your suggestions.